ee/tabby-webserver/src/service/db/users.rs

```diff
@@ -242,4 +290,226 @@ mod tests {
         assert_eq!(user.email, new_user.email);
         assert_ne!(user.auth_token, new_user.auth_token);
     }
+
+    #[tokio::test]
+    async fn test_list_users_with_filter() {
```

Comment:
is it possible to extract `list_<type>_with_filter` into a single utility so unit test doesn't have to run it separately for users and invitations?